test/conformance/device/device_adapter_hip.match

```diff
@@ -1,25 +1,6 @@
-{{OPT}}urDeviceCreateWithNativeHandleTest.Success
-{{OPT}}urDeviceGetTest.InvalidValueNumEntries
-{{OPT}}urDeviceGetGlobalTimestampTest.Success
-{{OPT}}urDeviceGetGlobalTimestampTest.SuccessSynchronizedTime
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_SINGLE_FP_CONFIG
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_DOUBLE_FP_CONFIG
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_QUEUE_PROPERTIES
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_IMAGE_SUPPORTED
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_MAX_READ_WRITE_IMAGE_ARGS
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_QUEUE_ON_DEVICE_PROPERTIES
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_QUEUE_ON_HOST_PROPERTIES
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_IL_VERSION
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_SUPPORTED_PARTITIONS
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_PARTITION_TYPE
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_GPU_EU_COUNT
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_GPU_EU_SIMD_WIDTH
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_GPU_EU_SLICES
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_GPU_SUBSLICES_PER_SLICE
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_GPU_HW_THREADS_PER_EU
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_MAX_MEMORY_BANDWIDTH
-{{OPT}}urDeviceGetInfoTest.Success/UR_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES
```

Comment:
Why is e.g. this case, UR_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES, removed when it is a supported query?

Reply:
These match files are lists of known failing tests. Removing a test from here means it should always be passing now.